File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q02.js
import { question } from "readline-sync"
function main(){
    //Input
    const value_hours = parseInt(question("Enter a value in hours: "))
    const value_minutes = parseInt(question("Enter a value in minutes: ")) 

    //Processing
    const total_value_minutes = value_hours * 60 + value_minutes

    //Output
    console.log(`The total value in minutes of ${value_hours} hours and ${value_minutes} minutes is: ${total_value_minutes} minutes`)
}


main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q02.py
def main():
    #Input
    value_hours = int(input("Enter a value in hours: "))
    value_minutes = int(input("Enter a value in minutes: "))

    #Processing
    total_value_minutes = (value_hours * 60) + value_minutes

    #Output
    print(f"Total value in minutes of {value_hours} hours and {value_minutes} minutes is: {total_value_minutes} minutes ")

main()